docs/the-new-architecture/landing-page.md

```diff
@@ -1,32 +1,72 @@
 ---
 id: landing-page
-title: Introduction
+title: About the New Architecture
 ---
 
-import NewArchitectureWarning from '../\_markdown-new-architecture-warning.mdx';
+In 2018, the React Native team began work to rewrite the internals of React Native to address design limitations that became apparent after years of usage. In 2024, the rewrite has been rolled out and proven at scale across React Native apps at Meta.
 
-<NewArchitectureWarning/>
+The team is now working to make this the default experience for the React Native open source ecosystem. The term _New Architecture_ refers to both the new framework architecture and the work to bring it to open source.
 
-Starting from version 0.68, React Native provides the New Architecture, which offers developers new capabilities for building highly performant and responsive apps. Visit [Why a New Architecture](why) to learn more about what drove the decision to re-architect and the benefits it provides.
+The New Architecture has been available for experimental opt-in as of [React Native 0.68](/blog/2022/03/30/version-068#opting-in-to-the-new-architecture) with continued improvements. The remaining work of the rollout is to ensure that the thousands of React Native libraries that developers depend on work out-of-the-box or have a straightforward migration path.
 
-To achieve these benefits, we had to rethink how Native Modules and Native Components work. This led us to develop the [Pillars of the New Architecture](pillars):
+## Why a New Architecture?
 
-- [The New Native Module System - Turbo Modules](pillars-turbomodules), a framework to support efficient and flexible integration with native code
-- [The New Native Renderer - Fabric](pillars-fabric-components), which offer improved capabilities, cross-platform consistency, and performance in rendering
-- [The Codegen](pillars-codegen), which generates boilerplate C++ required by the New Architecture via static typing in JavaScript
+Primarily, the New Architecture is an investment of the future of React Native to write polished app experiences. The New Architecture unlocks capabilities and improvements that were not possible in the legacy architecture.
 
-## Get Started with the New Architecture
+### Synchronous execution
```

Comment:
Can we include a code example with the before/after? I think that will make the value "click" better.